src/kv/RocksDBStore.cc

```diff
@@ -1306,21 +1312,6 @@ int64_t RocksDBStore::commit_cache_size()
   return total_bytes;
 }
 
-<<<<<<< HEAD
-int RocksDBStore::set_cache_high_pri_pool_ratio(double ratio)
-{
-  if (g_conf()->rocksdb_cache_type != "lru") {
-    return -EOPNOTSUPP;
-  }
-  dout(10) << __func__ << " old ratio: " 
-          << bbt_opts.block_cache->GetHighPriPoolRatio() << " new ratio: "
-          << ratio << dendl;
-  bbt_opts.block_cache->SetHighPriPoolRatio(ratio);
-  return 0;
-}
-
-=======
->>>>>>> src/kv: Initial import of a custom RocksDB cache.
```

Comment:
can you squash this part into the earlier commit that introduces it?

Reply:
ugh, that's irritating.  Let me see what I can do.